Rakefile

```diff
@@ -241,12 +241,12 @@ task xcode: [:dependencies] do
 end
 
 def fold(label)
-  puts "travis_fold:start:#{label}" if is_travis?
+  puts "travis_fold:start:#{label}" if travis?
   yield
-  puts "travis_fold:end:#{label}" if is_travis?
+  puts "travis_fold:end:#{label}" if travis?
 end
 
-def is_travis?
+def travis?
   !ENV['TRAVIS'].nil?
 end
```

Comment:
Out of scope for this PR, but we can remove any instance of Travis CI related code we run into, since it's no longer used.